src/main/java/org/entando/kubernetes/service/digitalexchange/crane/CraneCommand.java

```diff
@@ -67,6 +76,19 @@ public String getImageDigest(String image) {
         return outputLines.get(0);
     }
 
+    private static String readProcessOutput(ProcessHandler processHandler, boolean stderr, String fallback) {
+        String res;
+        try {
+            res = String.join("\n", (stderr)
+                    ? processHandler.getErrorLines()
+                    : processHandler.getOutputLines()
+            );
+        } catch (IOException e) {
+            res = fallback;
```

Comment:
we should log error at least at debug level

Reply:
btw this is old code refactore in a function